cypress/e2e/tests/migration/applicationinventory/applications/interlinked.test.ts

```diff
@@ -155,6 +156,56 @@ describe(["@tier3"], "Applications interlinked to tags and business service", ()
         cy.get(continueButton).click();
     });
 
+    it("Validates association application tags to  archetype tags ", function () {
+        //automates polarion MTA-401
+        //create archytype
```

Comment:
This comment, and other similars in this file are redundant, the code is explanatory enough

Reply:
Done!